cdap-watchdog/src/main/java/co/cask/cdap/logging/framework/CDAPLogAppender.java

```diff
@@ -84,7 +88,7 @@ public void setMaxFileLifetimeMs(long maxFileLifetimeMs) {
   public void start() {
     super.start();
     this.logFileManager = new LogFileManager(maxFileLifetimeMs, syncIntervalBytes, LogSchema.LoggingEvent.SCHEMA,
-                                             fileMetaDataManager, locationFactory);
+                                             fileMetaDataManager, locationFactory, cConfiguration);
```

Comment:
related to previous PR: `namespaceId.equals(NamespaceId.SYSTEM)` should be `namespaceId.equals(NamespaceId.SYSTEM.getEntityName())`